content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,98 @@ description: Upgrading an ambient mode installation with Helm.
 
-Node cordoning and blue/green node pools are recommended to control blast radius of application pod traffic disruption during production upgrades. See your Kubernetes provider documentation for details.
+Node cordoning and blue/green node pools are recommended to limit the blast radius of resets on application traffic during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
-## Prerequisites
+## Understanding ambient upgrades
+
+All Istio upgrades involve upgrading the control plane, data plane, and Istio CRDs. Because the ambient data plane is split across [two components](/docs/ambient/architecture/data-plane), the ztunnel and waypoints, upgrades involve separate steps for these components. Upgrading the control plane and CRDs is covered here in brief, but is essentially identical to [the process for upgrading these components in sidecar mode](docs/setup/upgrade/canary/).
 
-1. Update the Helm repository:
+Like sidecar mode, ambient mode upgrades can make use of [revision tags](https://istio.io/latest/docs/setup/upgrade/canary/#stable-revision-labels) to allow fine-grained control over ({{< gloss >}}gateway{{</ gloss >}}) upgrades, including waypoints, with simple controls for rolling back at any point. However, unlike sidecar mode, the ztunnel runs as a DaemonSet — a per-node proxy — meaning that ztunnel upgrades affect, at minimum, an entire node at a time. While this may be acceptable in many cases, applications with long-lived TCP connections may be disrupted.  In such cases, we recommend using node cordoning and draining before upgrading the ztunnel for a given node. For the sake of simplicity, this document will demonstrate in-place upgrades of the ztunnel, which may involve a short downtime.
+## Prerequisites
 
-    {{< text syntax=bash snip_id=update_helm >}}
-    $ helm repo update istio
-    {{< /text >}}
+### Organize your tags and revisions
 
-## In-place upgrade
+In order to safely upgrade a mesh in ambient mode, your gateways and namespaces should be using the `istio.io/rev` label to specify a revision tag which controls the version of the proxy that is running. We recommend dividing your production cluster into multiple tags to organize your upgrade. All members of a given tag will be upgraded simultaneously, so it is wise to begin your upgrade with your lowest risk applications. We do not recommend referencing revisions directly via labels for upgrades, as this process can easily result in the accidental upgrade of a large number of proxies, and is difficult to segment. To see what tags and revisions you are using in your cluster, see the section on upgrading tags.
```

Comment:
Agree with some other the other comments, we should not remove in-place upgrade. A good portion of our users are still using it for sidecars as long as they are okay with the limitations.

Reply:
Yes, John is going to add an in-place upgrade tab after this PR merges.  I don't have time to add that, as this PR has been in review for almost two months.

Reply:
Ok SG, you can ignore my new comment about in-place upgrade, before seeing yours. :)